GameServerLib/Logic/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -114,7 +114,7 @@ public virtual void Die(AttackableUnit killer)
                     return;
                 }
 
-                cKiller.OnKillMinion(this as Minion);
+                cKiller.OnKill(this);
```

Comment:
It will not return if killer is not a champion, and that will change the flow of code to change the IsDashing variable from true to false, and I'm not sure when it should be changed.

Reply:
I'd write this:
```cs
if ((killer != null) && (killer is Champion))
{
    ((Champion)killer).OnKill(this)
}
IsDashing = false;
```
Because you want to cancel dashes even when there is no ``Champion`` who killed him (using the death command for example, or being killed solely by minions).

Reply:
Ok, will change.